WeakAuras/WeakAuras.lua

```diff
@@ -2868,6 +2870,27 @@ function WeakAuras.Modernize(data)
     end
   end
 
+  -- Version 15 was introduced in April 2019
+  -- This adds a path to everything in authorOptions
+  -- so that we can locate it.
+  -- The other option is that we can build it locally each time
+  -- as transient data
+  if data.internalVersion < 15 then
```

Comment:
This data migration is, in principle, unnecessary. It is possible to construct transient path data from the database and use that instead, and save on the SV disk size. This would require a moderate restructuring of the codebase to handle it, so for the moment I think it is not worth it.

Reply:
Ah, this is a rebase error. I did not actually end up needing this migration.

Reply:
So the code can be removed?